lib/supplejack/item_relation.rb

```diff
@@ -66,5 +66,9 @@ def create(attributes = {})
     def method_missing(method, *args, &block)
       @items.send(method, *args, &block)
     end
+
+    def respond_to_missing?(_method, *_args, &_block)
```

Comment:
This `respond_to_missing?` returns `true` because the `method_missing` is not restricted it responds to any missing method call.